vardbg/output/video_writer/default_config.toml

```diff
@@ -45,3 +45,8 @@ heading_size = 24
 # Intro font, used to render the intro screen
 intro = "$ASSETS/fonts/Inter-Bold.ttf"
 intro_size = 48
+
+[theme]
+# Color scheme to use
+# Available Themes = (default, emacs, friendly, colorful, autumn, murphy, manni, monokai, perldoc, pastie, borland, trac, native, fruity, bw, vim, vs, tango, rrt, xcode, igor, paraiso-light, paraiso-dark, lovelace, algol, algol_nu, arduino, rainbow_dash, abap, solarized-dark, solarized-light, sas, stata, stata-light, stata-dark)
```

Comment:
```suggestion
# Available themes:
#   - default
#   - emacs
#   - friendly
#   - colorful
#   - ...
```

Packing so much information into a single line hurts readability — one theme per line takes up much more vertical space, but it makes this much more readable. Please also refrain from unnecessary capitalization and stick to normal language punctuation conventions here, since documentation is not code. It is important to consider context and readability when writing documentation.